docs/sources/mimir/references/http-api/index.md

```diff
@@ -425,6 +427,25 @@ GET /ingester/ring
 
 This endpoint displays a web page with the ingesters hash ring status, including the state, health, and last heartbeat time of each ingester.
 
+| [Ingester tenants](#ingester-tenants) | Ingester | `GET /ingester/tenants` |
+| [Ingester tenant TSDB](#ingester-tenant-tsdb) | Ingester | `GET /ingester/tsdb/{tenant}` |
```

Comment:
Should these be here?

Reply:
We document all endpoints, including those that render HTML pages. I followed example of store-gateway "tenant blocks" endpoints.

Reply:
These lines have been added twice though - once on lines 53-54, and here on lines 430-431.

Reply:
Oh 🤦 Now I'm glad that CI checks failed and auto-merge didn't proceed :) Thanks for catching that!